lib/gc/Transforms/Microkernel/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+gc_set_mlir_link_components(MLIR_LINK_COMPONENTS MLIRIR)
+
+include(onednn)
+
+get_property(GC_DNNL_INCLUDES GLOBAL PROPERTY GC_DNNL_INCLUDES)
+
+include_directories(${GC_DNNL_INCLUDES})
```

Comment:
Is there any reason to not make it additional headers for the dialect library?

Reply:
Somehow it doesn't take effect in additional headers... Changed it to `target_include_directories` to narrow down the scope.